flow/project.py

```diff
@@ -1217,6 +1219,8 @@ def __new__(metacls, name, bases, namespace):
         cls._GROUPS = []
         cls._GROUP_NAMES = set()
 
+        cls.operation_hook = cls._setup_hook_object(parent_class=cls)
```

Comment:
It seems to me that it is inconsistent to use the plural for project hooks and the singular for operation hooks.

Reply:
I think I addressed this in 594cdbf, but lmk if you had something else in mind